internal/access/identity.go

```diff
@@ -170,10 +170,6 @@ func UpdateIdentityInfoFromProvider(c RequestContext, oidc providers.OIDCClient)
 			logging.Errorf("failed to revoke invalid user session: %s", nestedErr)
 		}
 
-		if nestedErr := data.DeleteProviderUsers(db, data.ByIdentityID(identity.ID), data.ByProviderID(provider.ID)); nestedErr != nil {
```

Comment:
Removed this, on a failure the access is revoked, but we don't know that the user no longer exists. Its confusing when their group membership changes based on a failed login.